src/frontend/index.html

```diff
@@ -31,8 +31,8 @@
         <!-- NAV LINKS -->
 
         <nav>
-          <a href="login.html">
-            <button class="nav-button">
+          <a href="/login">
```

Comment:
Question about this: does it not need to point at the URL for the PHP server running in the docker container and/or whatever URL Seneca will give us in production?

That is to say, should this not somehow be read from the environment?  Or am I wrong?  What lives at `/login`?